vizro-core/src/vizro/_vizro.py

```diff
@@ -76,6 +76,18 @@ def _pre_build():
             if hasattr(model, "pre_build"):
                 model.pre_build()
 
+    @staticmethod
+    def _clear_state():
+        """Private method that clears all state in the vizro package."""
```

Comment:
```suggestion
        """Private method that clears all states in the vizro app."""
```

Reply:
Do yo think it's confusing because we also include the `data_manager` and `model_manager` here? Not sure if they really count as being part of the app 🤔 ?

Reply:
For me they are part of the app as otherwise the app wouldn't run. I was a bit confused about the package term because I wouldn't know what clearing the state for a python package means 🤔